### ISSUE TITLE
covid19-chatbot review

### ISSUE THREAD

**Issue by anitadewitt-va** [department-of-veterans-affairs/va.gov-team#20606]
## Instructions

1. Click or press the gear icon next to the "Labels" heading on the right. Search for your team in the labels list. If your team is not listed in the Labels menu, please leave a comment after you have created this issue. The Product Support team will create a new label for your team.
2. If this issue is being opened by a VFS team, please add the `vsp-product support` label.
3. **Issue tickets should be tested when you have created a pull request.** This gives the 508, or quality assurance, person time to review your code before it is merged into the `master` branch. Once code is merged into `master` it will be deployed to Staging. Reviewing and correcting issues becomes more difficult at this point.
4. Text inside `<-- comment -->` blocks will not appear in your issue ticket. These comments offer guidance on what information you should provide for each section.
5. Leave a link here to the 508 defect severity level: 

    - [508-defect-0 :exclamation: Launchblocker](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-0)
    - [508-defect-1 :exclamation: Launchblocker](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-1)
    - [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)
    - [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)
    - [508-defect-4](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-4)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** Anita DeWitt or Shefali Nayak 

## User Story or Problem Statement

There were two changes made to the coronavirus chatbot recently.
1. Fixing the autoscrolling issue 
2. Removal of timestamps from the message boxes 

## Details

**Fixing the autoscroll issue:**

Typically, after selecting a topic the screen autoscrolls to the user's response, however, this feature was broken. After our fix, we see the screen autoscroll and the user’s response is highlighted and there seem to be no issues when testing with a screen reader, but we would like another set of eyes on this issue. 

**Removal of timestamps from the message boxes:**

After screen reader testing and axe checks, we noticed that the timestamps for each message did not have an appropriate color contrast for users. We decided to remove this, but would like to check to make sure that removing this feature does not effect the accessibility of the bot. 

## Acceptance Criteria
- [ ] All messages within the chatbot are read out correctly from the screenreader 
- [ ] Autoscroll scrolls to the user's response 
- [ ] There are no color contrast issues within the bot 

## Environment
This can be found on all environments on va.gov at [https://www.va.gov/coronavirus-chatbot/](https://www.va.gov/coronavirus-chatbot/)

## Steps to Recreate
1. Navigate to [https://www.va.gov/coronavirus-chatbot/](https://www.va.gov/coronavirus-chatbot/)
2. Select a topic such as "COVID-19 prevention"
3. Verify that it scrolls to the user's response: "COVID-19 prevention" on the right side 
4. Redo steps 2 and 3 with a screen reader
5. Perform axe checks

**Comment by anitadewitt-va** [comment]
There is no label for the covid19-chatbot.
